src/containers/community/CommunityPosts.js

```diff
@@ -42,10 +43,10 @@ const getFeedModule = (community, currentUser, moduleChoice) => {
 
   if (showPopularSkills && showPostPrompt) {
     module.component = moduleChoice
-    ? <PopularSkillsModule community={community} />
+    ? <PopularSkillsModule slug={community.slug} />
```

Comment:
because this is getting called in componentDidMount, to avoid the server rendering warning, this community doesn't get updated when the fetchPopularSkills graphql query returns. So the popular skills are never shown.